cristin-project/src/test/java/no/unit/nva/cristin/projects/ContributorRoleMappingTest.java

```diff
@@ -6,26 +6,12 @@
 import java.util.NoSuchElementException;
 
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class ContributorRoleMappingTest {
 
-    @Test
-    void shouldReturnManagerMapping() {
-        assertTrue(ContributorRoleMapping.getNvaRole("PRO_MANAGER").get().equals("ProjectManager"));
-        assertTrue(ContributorRoleMapping.getCristinRole("ProjectManager").get().equals("PRO_MANAGER"));
-    }
-
-    @Test
-    void shouldReturnParticipantMapping() {
-        assertTrue(ContributorRoleMapping.getNvaRole("PRO_PARTICIPANT").get().equals("ProjectParticipant"));
-        assertTrue(ContributorRoleMapping.getCristinRole("ProjectParticipant").get().equals("PRO_PARTICIPANT"));
-    }
-
     @Test
     void shouldFailWhenUnknownRole() {
         assertThrows(NoSuchElementException.class,
```

Comment:
This isn't tested in the API?

Reply:
Not yet... It will be testet in the next PR regaring create project... Illegal input to create method called from GUI will result in 400 Bad Request. When reading from cristin errors will be ignored and logged.